pkg/api/kptfile/v1alpha2/validation_test.go

```diff
@@ -276,7 +276,7 @@ func TestValidatePath(t *testing.T) {
 		},
 		{
 			"./*",
-			true,
+			false,
 		},
```

Comment:
we should have a test case for a path like `a/../../b`

Reply:
Added.